packages/suite/src/components/guide/Content.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+    height: 100%;
+    overflow-y: auto;
+    padding: 0 22px;
+`;
+
+interface Props {
+    children: React.ReactNode;
+}
+
+const Content = ({ children, ...rest }: Props) => <Wrapper {...rest}>{children}</Wrapper>;
```

Comment:
minor:
React.FC type is useful if you want to allow children for a Function Component. 
```suggestion
const Content: : React.FC = ({ children, ...rest }) => <Wrapper {...rest}>{children}</Wrapper>;
```

Reply:
On a second view: there is no point in splitting children from the rest of the props, it's just a regular prop. You could export the Wrapper directly...